### PR TITLE
Update Edge prerequisites for K8s 1.22

### PR DESCRIPTION
**Pull request by matthijskrul**
## Summary of the changes / Why this is an improvement
- Updates the prerequisites for CrateDB Edge installation in the tutorial for the new support for K8s 1.22

## Checklist

 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/edge.rst [hunk 1]
```diff
@@ -65,17 +65,31 @@ Kubernetes cluster, one that meets the following requirements:
...
+  at least 4 CPU cores for reliable performance) as well as sufficient compute
+  to run the CrateDB instances desired. We recommend the use of a K8s
+  autoscaler.
 
-* A Kubernetes version > 1.15 and < 1.22 (support for Kubernetes versions 1.22+
-  is coming in the near future);
+* A Kubernetes version 1.20, 1.21, or 1.22;
 
-* A Kubernetes load balancer for accessing CrateDB Clusters;
+* A Kubernetes load balancer for accessing CrateDB clusters;
```
Comment (WalBeh) [inline docs/edge.rst]: We do **NOT** require a loadbalancer for accessing crateDB as we are using _cloudflare tunnels_ to work around networking issues.

File: docs/edge.rst [hunk 2]
```diff
@@ -65,17 +65,31 @@ Kubernetes cluster, one that meets the following requirements:
...
 
-* A storage class for persistent data.
+* A storage class for persisting your data (SSD recommended). As outlined in
+  the installation script, CrateDB Edge expects the storage classes
+  ``crate-premium`` (SSD) and ``crate-standard`` (SSD or Spindle). Also ensure
+  that you are able to set the field ``allowVolumeExpansion`` to ``true``.
+
+* Unless you are experienced with operating Kubernetes clusters, we recommend
+  to start with a dedicated Kubernetes cluster for CrateDB Edge. If you use an
+  existing Kubernetes setup, be aware that the following components installed
```
Comment (WalBeh) [inline docs/edge.rst]:
```suggestion
  existing Kubernetes setup, be aware that the following components will be setup
```

File: docs/edge.rst [hunk 3]
```diff
@@ -65,17 +65,31 @@ Kubernetes cluster, one that meets the following requirements:
...
+  during the installation of CrateDB Edge may interfere with your existing
```
Comment (WalBeh) [inline docs/edge.rst]:
```suggestion
  during the installation of CrateDB Edge which may interfere with your existing
```

File: docs/edge.rst [hunk 4]
```diff
@@ -65,17 +65,31 @@ Kubernetes cluster, one that meets the following requirements:
...
+  configuration:
+
+  * cert-manager (version 1.6.1)
+  * ingress-nginx (version 1.2.3)
+  * Optionally, Grafana, Loki, or Promtail.
```
Comment (WalBeh) [inline docs/edge.rst]:
```suggestion
  * Optionally, Grafana, Loki, or Promtail (via helm)
```

File: docs/edge.rst [hunk 5]
```diff
@@ -65,17 +65,31 @@ Kubernetes cluster, one that meets the following requirements:
...
   however, that you will only be able to provision single-node CrateDB
-  "clusters";
+  clusters;
 
-* Sufficient CPU per node to run the CrateDB Cloud software stack and the OS
-  (we recommend at least 4 CPU cores for reliable performance);
+* Sufficient CPU per node to run the CrateDB Cloud software stack (we recommend
+  at least 4 CPU cores for reliable performance) as well as sufficient compute
+  to run the CrateDB instances desired. We recommend the use of a K8s
+  autoscaler.
```
Comment (WalBeh) [inline docs/edge.rst]:
```suggestion
  autoscaler. We "request" about 2 CPUs and 2500Mi  of memory for our setup (without grafana & loki).
```

**Review by WalBeh — APPROVED** [review]

**Comment by matthijskrul** [comment]
@WalBeh thanks for the input!

**Pull request merged**